app/models/competition.rb

```diff
@@ -2208,9 +2208,29 @@ def find_round_for(event_id, round_type_id, format_id = nil)
     end
   end
 
-  def dues_per_competitor_in_usd
-    dues = DuesCalculator.dues_per_competitor_in_usd(self.country_iso2, self.base_entry_fee_lowest_denomination.to_i, self.currency_code)
-    dues.presence || 0
+  def export_for_dues_generation
+    num_competitors = competitors.count
```

Comment:
I think this will generate one extra query per row. Why did you get rid of the `COUNT(DISTINCT persons.id)` in the `select_attributes` of the controller? Did I tell you that during review? :sweat_smile: